static/scenes/railroad/js/level.js

```diff
@@ -40,7 +40,7 @@ class Level {
     const intersections = this.raycasterSystem.cast(clickEvent);
 
     if (intersections.length > 0) {
-      this.presentSystem.shoot(intersections[0].point);
+      this.presentSystem.shoot(intersections[0].point, intersections[0].object);
```

Comment:
An idea to separate present logic from elf logic: Do the check to see if the target is an elf here (we'll need to do other checks for the target being an elf here anyway) and then pass in an optional function to be called to the present system when the present lands, and have that function updates the elf sprite.